core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -273,7 +273,8 @@ object TestUtils extends Logging {
                          logDirCount: Int = 1,
                          enableToken: Boolean = false,
                          numPartitions: Int = 1,
-                         defaultReplicationFactor: Short = 1): Properties = {
+                         defaultReplicationFactor: Short = 1,
+                         processRoles: String = ""): Properties = {
```

Comment:
Adding function args here is a no no. Too many downstream dependencies. Since we have only one usage at the moment, it is better to modify the `Properties` that is returned from `createBrokerConfig`. We can think of something nicer when we have more uses.